examples/sample-app/README.md

```diff
@@ -187,11 +210,15 @@ the present working directory is the same directory as this README.
 
         $ https://<host>:8443/osapi/v1beta1/buildConfigHooks/ruby-sample-build/secret101/github?namespace=test
 
+	**VAGRANT USERS**:  Skip this step, as webhooks will not be able to reach the OpenShift system.
```

Comment:
It doesn't make sense to write it here, since the next note specifically says your OpenShift server must by publicly accessible and it applies to both running inside vagrant as well as on your machine.